calendar-bundle/src/Resources/contao/models/CalendarEventsModel.php

```diff
@@ -207,7 +207,7 @@ public static function findPublishedByParentAndIdOrAlias($varId, $arrPids, array
 		if (!static::isPreviewMode($arrOptions))
 		{
 			$time = \Date::floorToMinute();
-			$arrColumns[] = "($t.start='' OR $t.start<='$time') AND ($t.stop='' OR $t.stop>'" . ($time + 60) . "') AND $t.published='1'";
+			$arrColumns[] = "$t.published='1' AND ($t.start='' OR $t.start<='$time') AND ($t.stop='' OR $t.stop>'$time')";
```

Comment:
Just in case you wonder, I have adjusted the order to be forward compatible with Contao 4.10.